robottelo/common/helpers.py

```diff
@@ -371,3 +388,28 @@ def read_data_file(filename):
     absolute_file_path = get_data_file(filename)
     with open(absolute_file_path, 'r') as file_contents:
         return file_contents.read()
+
+
+def configure_entities():
+    """Set :data:`robottelo.orm.DEFAULT_SERVER_CONFIG`.
+
+    Set :data:`robottelo.orm.DEFAULT_SERVER_CONFIG` to whatever is returned by
+    :meth:`robottelo.common.helpers.get_nailgun_config`. See
+    :class:`robottelo.orm.Entity` for more information on the effects of this.
+
+    Emit a warning and do not set ``robottelo.orm.Entity._server_config`` if no
+    ``robottelo.properties`` configuration file is available.
+
+    """
+    try:
+        orm.DEFAULT_SERVER_CONFIG = get_nailgun_config()
+    except KeyError:
+        warn(
```

Comment:
Why not logging.warn? Even though all warnings will be sent to logging because the Robottelo's logging configuration